### PR TITLE
validation-service stuff

### PR DESCRIPTION
**Pull request by hawkaa**
1. Docs
2. Max size for buildings
3. Building buffers

### CODE REVIEW

File: platform-services/validation-service/README.md [hunk 1]
```diff
@@ -16,7 +19,20 @@ Buildings are represented in this JSON data structure:
...
-## `POST /validate-buildings`
+## Validation rules
+
+The validation service will validate that:
+
+1. All buildings are within bounds, which means `x` and `y` are greater than 0,
+   `x + dx` less than 50 and `y + dy` less than 100.
+2. All buildings have a size larger than 0, that is `dx` and `dy` are larger than 0.
+3. Buildings are no taller than 9, that is `dz` smaller than 10.
+4. Buildings are no larger than 19, that is `dx` and `dz` are smaller than 20.
```
Comment (simehe) [inline platform-services/validation-service/README.md]: You mean dx and dy?

File: platform-services/validation-service/README.md [hunk 2]
```diff
@@ -16,7 +19,20 @@ Buildings are represented in this JSON data structure:
 }
 ```
 
-## `POST /validate-buildings`
+## Validation rules
+
+The validation service will validate that:
+
+1. All buildings are within bounds, which means `x` and `y` are greater than 0,
+   `x + dx` less than 50 and `y + dy` less than 100.
```
Comment (simehe) [inline platform-services/validation-service/README.md]: Looking through the current network calls to https://abakus.spacemaker.ai, x + dx <= 100 and y + dy <= 50.

File: platform-services/validation-service/README.md (hunk 1 above)
Comment (hawkaa) [reply platform-services/validation-service/README.md]: yes, thank you.

File: platform-services/validation-service/README.md (hunk 2 above)
Comment (hawkaa) [reply platform-services/validation-service/README.md]: right!

**Comment by hawkaa** [comment]
Thanks @simehe ! Will fix!

File: platform-services/validation-service/README.md (hunk 2 above)
Comment (hawkaa) [reply platform-services/validation-service/README.md]: They are zero indexed, thus we want y to be >= 0 and < 50, thus making 0 the lowest value and 49 the highest value.

**Comment by hawkaa** [comment]
@simehe how does it look like now?

**Pull request merged**